python/langsmith/client.py

```diff
@@ -3369,6 +3370,133 @@ def create_example_from_run(
             created_at=created_at,
         )
 
+    def upsert_examples_multipart(
+        self,
+        *,
+        upserts: List[ls_schemas.ExampleCreateWithAttachments] = [],
```

Comment:
should not initialize this to empty list due to the way python handles mutable default arguments, see here https://nikos7am.com/posts/mutable-default-arguments/

to fix, just remove the default